app/controllers/api/storage.php

```diff
@@ -21,8 +22,106 @@
 use Appwrite\OpenSSL\OpenSSL;
 use Appwrite\Utopia\Response;
 use Utopia\Config\Config;
+use Utopia\Validator\Integer;
+use Appwrite\Database\Exception\Authorization as AuthorizationException;
+use Appwrite\Database\Exception\Structure as StructureException;
 use Utopia\Database\Query;
-use Utopia\Validator\Numeric;
+
+App::post('/v1/storage/buckets')
+    ->desc('Create storage bucket')
+    ->groups(['api', 'storage'])
+    ->label('scope', 'buckets.write')
+    ->label('event', 'storage.buckets.create')
+    ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_JWT])
+    ->label('sdk.namespace', 'storage')
+    ->label('sdk.method', 'createBucket')
+    ->label('sdk.description', '/docs/references/storage/create-bucket.md')
```

Comment:
We also need to create this description file, which should follow the same line of other similar routes.

Reply:
done